test/integration/expression/tests/zoom/interpolate-projection/test.json

```diff
@@ -18,9 +18,9 @@
       "zoom"
     ],
     0,
-    ["literal", ["vertical-perspective", "vertical-perspective", 1]],
+    "vertical-perspective",
```

Comment:
Does this mean literal is no longer supported?

Reply:
Yes, it makes very little sense to me why we want to support literals, unless we want to allow transitions from one intermediate:

 `["vertical-perspective", "mercator", 0.2]` 
 
 to another
 
  `["equal-earth", "utm23", 0.5]` 
  
  which I think is _too_ flexible

Reply:
Fair enough.